src/index.js

```diff
@@ -3,7 +3,7 @@ import * as t from "@babel/types";
 
 // this is the code that we will generate for Pressability
 const _onFsPressForward_PressabilityCode = `_onFsPressForward_Pressability = function(isLongPress) {
-  if (UIManager.onFsPressForward == null) {
+  if (UIManager == null || UIManager.onFsPressForward == null) {
```

Comment:
This works, but it's because of two different gotchas canceling each other out. In JS, `null` and `undefined` are two different things. From the screenshots of the issue, I believe UIManager was `undefined`. But using `==` allows for type coercion, so JS it considers them equal here. 

`===` is generally recommended for comparisons because it doesn't do type coercion, but it would return false when comparing `undefined` to `null`.

However, in this case, we don't want `null` or `undefined`, so I would probably rewrite it like so:
```suggestion
  if (!UIManager || !UIManager.onFsPressForward) {
```

It is effectively the same check, but makes it more clear that you're avoiding any "falsey" value (`undefined`, `null`, etc.) rather than `null` specifically.

Reply:
> This works, but it's because of two different gotchas canceling each other out. In JS, `null` and `undefined` are two different things. From the screenshots of the issue, I believe UIManager was `undefined`. But using `==` allows for type coercion, so JS it considers them equal here.
> 
> `===` is generally recommended for comparisons because it doesn't do type coercion, but it would return false when comparing `undefined` to `null`.
> 
> However, in this case, we don't want `null` or `undefined`, so I would probably rewrite it like so:
> 
> It is effectively the same check, but makes it more clear that you're avoiding any "falsey" value (`undefined`, `null`, etc.) rather than `null` specifically.

Yep, I was expecting the `== null` to both null check and do the coercion if undefined, and if it is more idiomatic to use `!`, then I agree with your suggested changes.